content/enterprise_influxdb/v1.9/administration/monitor/logs.md

```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
 
 ## Tracing
 
-Logging has been enhanced, starting in InfluxDB 1.5, to provide tracing of important InfluxDB operations. Tracing is useful for error reporting and discovering performance bottlenecks.
+Logging has been enhanced to provide tracing of important InfluxDB operations.
+Tracing is useful for error reporting and discovering performance bottlenecks.
 
-See [Tracing](/enterprise_influxdb/v1.9/administration/logs/#tracing) in the InfluxDB OSS documentation.
+### Logging keys used in tracing
+
+#### Tracing identifier key
+
+The `trace_id` key specifies a unique identifier for a specific instance of a trace.
+You can use this key to filter and correlate all related log entries for an operation.
+
+All operation traces include consistent starting and ending log entries, with the same message (`msg`) describing the operation (e.g., "TSM compaction"), but adding the appropriate `op_event` context (either `start` or `end`).
+For an example, see [Finding all trace log entries for an InfluxDB operation](#finding-all-trace-log-entries-for-an-influxdb-operation).
+
+**Example:** `trace_id=06R0P94G000`
+
+#### Operation keys
+
+The following operation keys identify an operation's name, the start and end timestamps, and the elapsed execution time.
+
+##### `op_name`
+Unique identifier for an operation.
+You can filter on all operations of a specific name.
+
+**Example:** `op_name=tsm1_compact_group`
+
+##### `op_event`
+Specifies the start and end of an event.
+The two possible values, `(start)` or `(end)`, are used to indicate when an operation started or ended.
+For example, you can grep by values in `op_name` AND `op_event` to find all starting operation log entries.
+For an example of this, see [Finding all starting log entries](#finding-all-starting-operation-log-entries).
+
+**Example:** `op_event=start`
+
+##### `op_elapsed`
+Amount of time the operation spent executing.
+Logged with the ending trace log entry.
+Time unit displayed depends on how much time has elapsed -- if it was seconds, it will be suffixed with `s`.
+Valid time units are `ns`, `µs`, `ms`, and `s`.
+
+**Example:** `op_elapsed=0.352ms`
```

Comment:
This doesn't look like a valid duration (as Flux/InfluxDB generally parses them, but that may be something unique to the tracing data format. I'm not sure. I would expect this to be `op_elapsed=352ms` or `op_elapsed=352µs` (the equivalent of `0.325ms`). Duration values typically don't support fractional numbers.